src/index.ts

```diff
@@ -94,7 +94,10 @@ function escape(str: string) {
  * Interpolate the `&` with style name.
  */
 function interpolate(selector: string, styleName: string) {
-  return selector.replace(/&/g, styleName);
+  return styleName
+    .split(/, */)
```

Comment:
The issue is it needs a proper parser because it’s valid to have a comma in a CSS selector, I.e. the selector is escaped or part of a content string or some other match. Although unlikely, it should be handled.

Reply:
Makes sense. I was afraid of that -- a parser is a whole different kind of lift. Thanks for the reply, I'll give it some more thought.